core/include/keyman/keyman_core_api.h

```diff
@@ -310,6 +310,38 @@ km_core_context_items_to_utf8(km_core_context_item const *item,
                               char *buf,
                               size_t *buf_size);
 
+/*
+```
+### `km_core_context_items_to_utf32`
+##### Description:
+Convert a context item array into a UTF-32 encoded string placing it into
+the supplied buffer of specified size, and return the number of codepoints
+actually used in the conversion. If null is passed as the buffer the
+number codepoints required is returned. This will strip markers from the
+context during the conversion.
```

Comment:
```suggestion
Convert a context item array into a UTF-32 encoded string placing it into
the supplied buffer of specified size, and return the number of codepoints
actually used in the conversion. If null is passed as the buffer, the
number of codepoints required is returned.
Any markers in the context will not be included in the output buffer.
```

I assume it doesn't actually mutate the context _array_, just the output. Also, typo.

Reply:
Thank you will apply this